packages/core/src/PlayerContextProvider.js

```diff
@@ -345,7 +336,8 @@ export class PlayerContextProvider extends Component {
     // if not, then load the first track in the new playlist, and pause.
     return {
       ...baseNewState,
-      ...getGoToTrackState({ prevState, index: 0, shouldPlay: false })
+      ...getGoToTrackState({ prevState, index: 0, shouldPlay: false }),
+      mediaCannotPlay: false
```

Comment:
This looks good. Can we also set `mediaCannotPlay: false` above when `!isPlaylistValid(newPlaylist)`?

Reply:
Just kidding ignore this - I realized that's part of the `defaultState`.